include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -28,6 +28,11 @@ def RegisterOp : CalyxPrimitive<"register", [
       auto i1Type = $_builder.getI1Type();
       auto widthType = $_builder.getIntegerType(width);
       $_state.addTypes({widthType, i1Type, i1Type, i1Type, widthType, i1Type});
+    }]>,
+    OpBuilder<(ins "StringAttr":$name, "Type":$type), [{
```

Comment:
Oops, this addition snuck in. Let me know if it makes sense to split out or just include here. It seemed useful to me when creating registers from C++.